### DISCUSSION TITLE
Export individual entries to image format for easy sharing with others

### DISCUSSION THREAD

**Discussion by dongrentianyu** [TiddlyWiki/TiddlyWiki5#7335] [Ideas]
Currently, exporting individual entries can only be done as tid files, or csv files, or json files. But this is not conducive to w sharing for others to see directly. It would be great if we could add exporting to image format for sharing to others.

**Comment by pmario** [comment]
What do you mean by "image format"? You want to create screenshots of rendered pages, or do you want to export JPG images as single files?

**Reply by dongrentianyu** [reply]
I want to export individual entries as image files, not hmtl format files.
